kuma/static/styles/minimalist/components/_main-nav.scss

```diff
@@ -96,4 +109,8 @@
             }
         }
     }
+
+    &.show {
```

Comment:
I think you need to add something to override the `display: block` on tablet and up. If I open the menu and resize, it looks like this:
<img width="1254" alt="Screen Shot 2020-05-20 at 3 13 04 PM" src="https://user-images.githubusercontent.com/650/82487536-7a972680-9aac-11ea-8e00-ba67f54c7da6.png">

Reply:
Hmmm, strange. I have not seen this. Will take a look.

Reply:
Now I understand what you mean! Nice catch @mindy :tea:  Fixed.